examples/openvino/resnet50/user_script.py

```diff
@@ -0,0 +1,263 @@
+# -------------------------------------------------------------------------
```

Comment:
Where is this script used? I saw only `imagenet.py` is referenced in the config file.